autometa/common/external/samtools.py

```diff
@@ -37,22 +56,31 @@ def run(cmd):
     logger.debug(f'cmd: {cmd}')
     with open(os.devnull, 'w') as stdout, open(os.devnull, 'w') as stderr:
         retcode = subprocess.call(cmd, stdout=stdout, stderr=stderr, shell=True)
-    if retcode:
+    if not retcode:
+        return True
+    else:
         logger.warning(f'args:{cmd} retcode:{retcode}')
         return False
-    return True
 
-def sort(sam, out, nproc=0):
+def sort(sam, out, nproc=mp.cpu_count()):
+    """
+    Views the sam file and then sorts the alighnments by leftmost coordinates.
+    
+    Parameters
+    ----------
+    sam : str
+        </path/to/alignment.sam>
+    out : str
+        </path/to/output/file
+    nproc : int, optional
```

Comment:
Great catch on adding `optional`. I take it you read through the numpy [docstring guide](https://numpydoc.readthedocs.io/en/latest/format.html)? This should probably be done throughout the code base. I was not aware of this previously. 💯

Reply:
Actually no, I just used VS code extension for docstring and just to be consistent with your code I used the numpy format. I can read through the guide now

Reply:
Well, in any case. The VS code formatted it appropriately. 👍